packages/playground/wordpress/src/mu-plugins/auto_login.php

```diff
@@ -1,59 +1,86 @@
 <?php
 /**
- * Automatically logs the user in to aid the `login` Blueprint step and 
+ * Automatically logs the user in to aid the `login` Blueprint step and
  * the Playground runtimes.
  *
  * There are two ways to trigger the auto-login:
- * 
+ *
  * ## The `PLAYGROUND_AUTO_LOGIN_AS_USER` constant
  *
  * Used by the `login` Blueprint step does.
- * 
+ *
  * When the `PLAYGROUND_AUTO_LOGIN_AS_USER` constant is defined, this mu-plugin
  * will automatically log the user in on their first visit. The username is
  * the value of the constant.
- * 
+ *
  * On subsequent visits, the `playground_auto_login_already_happened` cookie will be
  * detected and the user will not be logged in. This means the "logout" feature
  * will work as expected.
- * 
+ *
  * ## The `playground_force_auto_login_as_user` GET parameter
- * 
+ *
  * Used by the "login" button in various Playground runtimes.
- * 
+ *
  * Only works if the `PLAYGROUND_FORCE_AUTO_LOGIN_ENABLED` constant is defined.
- * 
+ *
  * When the `playground_force_auto_login_as_user` GET parameter is present,
  * this mu-plugin will automatically log in any logged out visitor. This will
  * happen every time they visit, not just on their first visit.
- * 
- * 
+ *
+ *
  * ## Context
- * 
+ *
  * The `login` step used to make a HTTP request to the `/wp-login.php` endpoint,
  * but that approach had significant downsides:
- * 
+ *
  * * It only worked in web browsers
  * * It didn't support custom login mechanisms
  * * It required storing plaintext passwords in the Blueprint files
  */
 
+/**
+ * Returns the username to auto-login as, if any.
+ *
+ * @return string|false
+ */
+function get_playground_auto_login_as_user() {
```

Comment:
Let's prefer names that read like something a person would say, e.g. `playground_get_username_for_auto_login` (where `playground_` is a prefix to avoid function name collisions).